proposals/0000-accept-prs-on-github.rst

```diff
@@ -0,0 +1,89 @@
+.. proposal-number:: Leave blank. This will be filled in when the proposal is
+                     accepted.
+
+.. trac-ticket:: Leave blank. This will eventually be filled with the Trac
+                 ticket number which will track the progress of the
+                 implementation of the feature.
+
+.. implemented:: Leave blank. This will be filled in with the first GHC version which
+                 implements the described feature.
+
+.. highlight:: haskell
+
+Accept PRs on GitHub
+====================
+
+This proposes to accept pull requests (PRs) against GHC on GitHub (GH). There are a few
+possible mechanisms for doing so, as delineated below.
+
+This proposal comes directly out of a `conversation <https://mail.haskell.org/pipermail/ghc-devs/2016-September/012828.html>`_
+on the mailing lists.
+
+Motivation
+----------
+
+Lack of this feature to our workflow has been cited many times as a barrier to
+contributors. So let's do it!
+
+Proposed Change
+---------------
+
+* Start welcoming PRs on GH.
+
+From there, we have two choices:
+
+1. Automatically migrate all GH PRs to become Differentials in Phab. Users submitting
+   a PR on GH will be politely informed that their PR has been migrated to Phab, along
+   with instructions on how they can follow the PR.
+
+2. Create a tool (I will call it ``ghc-hub``) that GHC HQ (only) will use. This tool
+   allows migration of a GH PR to a Diff on Phab. GHC HQ will migrate "large" patches
+   (for an evolving definition of "large") to Phab and keep "small" patches on GH.
+   Generally, "small" patches will be accepted as-is; back-and-forth on a patch would
+   take place on Phab.
+
```

Comment:
I'm still not convinced that you'd want to do the initial diff (or do you intend to also run the `arc update`?)

Giving this a go (manually), for say 3 month (until 1.1.2017), and then see how things went sounds like a good approach to me. Clearly if PRs go out of hand too fast, we might need to reconsider earlier. Without any experience of what type of PRs and how many there are going to be, it's hard to tell. I'm particularly bad at crystal ball gazing.